src/common/records.lua

```diff
@@ -51,9 +51,23 @@ function records.getRecord(name)
 end
 
 --- Get all records from the ANT
----@return string The encoded JSON representation of all records.
+---@return table<table> The sorted records of the ANT
 function records.getRecords()
-	return json.encode(Records)
+	local antRecords = utils.deepCopy(Records)
+	assert(antRecords, "Failed to copy Records")
+	local recordEntries = utils.entries(antRecords)
+	-- sort the records alphabetically, ensuring "@" record is first
+	table.sort(recordEntries, function(a, b)
+		if a[1] == "@" then
+			return true
+		end
+		if b[1] == "@" then
+			return false
+		end
+		return a[1] < b[1]
+	end)
+
+	return recordEntries
```

Comment:
this should just be a flat array with the record details and key

```
[
{ 
  "record": "@",
   "transactionId: "xxxx",
    "ttlSeconds": 9000
 }
]
```
ANTs can choose how they'll sort this array. by default we can do '@' first and the rest alphabetically.

Reply:
[e1f8b75](https://github.com/ar-io/ar-io-ant-process/pull/35/commits/e1f8b755a516bb24c8d746247fffb9355f182b87)